ocaml/xapi/repository.ml

```diff
@@ -147,7 +147,17 @@ let get_proxy_params ~__context repo_name =
   | _ ->
       ("", "", "")
 
-let sync ~__context ~self ~token ~token_id =
+let ext_host_verified_rpc ~__context ~cert host_address xml =
```

Comment:
what does `ext` mean? execute?

Reply:
external, host out of the pool.

Reply:
How about `verified_rpc`? 
BTW, the `xml` in a `RPC` seems confusing.

Reply:
It's fine, I was just asking :)